fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/ConsentService.java

```diff
@@ -35,4 +41,14 @@ public boolean confirmConsent(String authId, UUID xRequestId) {
         log.debug("consent confirmation response code: {}", statusCode);
         return statusCode.is2xxSuccessful();
     }
+
+    public void deleteConsent(UserEntity userEntity, ConsentType consentType, String bankID) {
+        Optional<ConsentEntity> optionalConsent = consentRepository.findByUserEntityAndBankIdAndConsentTypeAndConsentConfirmed(userEntity,
+                bankID, consentType, Boolean.TRUE);
+        if (optionalConsent.isPresent()) {
+            log.info("consent {} for user {} is deleted", consentType, userEntity.getLoginUserName());
```

Comment:
Misleading log message as for me - looks like entity is `already` deleted, while, in fact we are deleting it now